dockerfiles/nginx-tls/nginx.conf.tpl

```diff
@@ -29,7 +29,10 @@ http {
     ssl_ciphers         ECDHE-RSA-AES128-GCM-SHA256:ECDHE-RSA-AES256-GCM-SHA384:ECDHE-RSA-AES128-SHA256:ECDHE-RSA-AES256-SHA384:ECDHE-RSA-AES256-GCM-SHA512:DHE-RSA-AES256-GCM-SHA512:DHE-RSA-AES256-GCM-SHA384;
     ssl_certificate     /tmp/tls/cert.pem;
     ssl_certificate_key /tmp/tls/key.pem;
-    ssl_session_cache   shared:SSL:1m;
+    ssl_session_cache   shared:SSL:2m;
+
+    client_max_body_size    5m;
+    client_body_buffer_size 50k;
```

Comment:
```suggestion
    client_body_buffer_size 64k;
```
I suggest we use a nice round number to line up with memory page boundaries.  [The default value is two pages](http://nginx.org/en/docs/http/ngx_http_core_module.html#client_body_buffer_size) which is 8k or 16k depending on the system.

Reply:
The nginx config used in old Verify infrastructure uses 50k for client_body_buffer_size. The intention of this pull request is to make nginx config used in new Verify infrastructure to have same settings as nginx config used in old Verify infrastructure.

Reply:
Discussed in person, we're happy with 64k.

Reply:
Done.